### DISCUSSION TITLE
Is there a way to create a cross-fade carousel in daisyUI?

### DISCUSSION THREAD

**Discussion by ads1029** [saadeghi/daisyui#1264] [Q&A] [answered]
Tried to search for a few days but could not get what I need, looks like daisyUI carousel only contains left-right in-and-out animation? 
The link below is what I found in Bootstrap UI that achieves a similar result:  https://getbootstrap.com/docs/4.1/components/carousel/#crossfade
Also here is an issue with people try to create it with tailwind, but ended up some bug: https://stackoverflow.com/questions/71895945/trying-to-use-a-cross-fade-carousel-in-tailwindcss-but-my-images-just-keep-staki

Any suggestion would be appreciated, cheers!

**Comment by saadeghi** [comment, marked as answer]
daisyUI is CSS-only and its carousel is just a basic CSS-only implementation. Honestly I added this component just to explore the possibilities but to be fair, it can not compete with JS plugins at all.  
I'm not saying it's impossible to do any other carousel effects with CSS but don't expect much from the carousel.
I suggest If you need a more advance carousel, use a JS plugin (or write your own). There are plenty of them for any framework.  
You can only use the daisyUI components that you need and use JS for the things that are better with JS (carousels, datepickers, modals, etc.)  

Let me know if you have any questions.

**Reply by ads1029** [reply]
Thanks a lot for the explanation! All clear :)
